packages/fast-usdc/test/cli.test.ts

```diff
@@ -0,0 +1,15 @@
+import test from 'ava';
+import { execa } from 'execa';
+import { fileURLToPath } from 'url';
+import { dirname, join } from 'path';
+
+const dir = dirname(fileURLToPath(import.meta.url));
+const CLI_PATH = join(dir, '../src/cli.js');
+
+test('CLI shows help when run without arguments', async t => {
+  const { stderr } = await execa('node', [CLI_PATH], {
```

Comment:
`execa` is pretty heavy-weight. I see lots of stuff added to `yarn.lock`. I suspect it won't cost more than 10 lines to stick with node's stdlib. Shall we try it?